frontend/src/components/profile/TrainingTips.tsx

```diff
@@ -0,0 +1,169 @@
+import { icons } from '@/style/Icon';
+import {
+    Box,
+    Button,
+    Card,
+    CardActionArea,
+    CardContent,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    Grid2,
+    Stack,
+    SvgIconProps,
+    SvgIconTypeMap,
+    Typography,
+} from '@mui/material';
+import { OverridableComponent } from '@mui/material/OverridableComponent';
+import { useRouter } from 'next/navigation';
+import { ReactNode, useState } from 'react';
+
+export function TrainingTipsButton() {
+    const [showDialog, setShowDialog] = useState(false);
+
+    return (
+        <>
+            {showDialog && (
+                <TrainingTipsDialog
+                    closeDialog={() => setShowDialog(false)}
+                    open={showDialog}
+                    onClose={() => setShowDialog(false)}
```

Comment:
Why do we have both `closeDialog` and `onClose` with the exact same implementation?